### PR TITLE
Try using jsonp in frontend instead of asking server

### PR DESCRIPTION
**Pull request by JavierJia**
@JeremyLi28 
I just give a try, it seems working 😄

### CODE REVIEW

File: neo/app/assets/javascripts/sidebar/controllers.js [hunk 1]
```diff
@@ -29,16 +29,9 @@ angular.module('cloudberry.sidebar', ['cloudberry.common'])
...
-        $.ajax({
-          url: url,
-          dataType: "json",
-          success: function (data) {
-            $('#tweet').append(data.html);
-          },
-          error: function(data) {
-            // do nothing
-          }
+        var url = "https://api.twitter.com/1/statuses/oembed.json?callback=JSON_CALLBACK&id=" + d.tid
```
Comment (JeremyLi28) [inline neo/app/assets/javascripts/sidebar/controllers.js]: what is the callback for? I think this is the only difference between our approach

**Comment by JeremyLi28** [comment]
Cool! Let me check out

File: neo/app/assets/javascripts/sidebar/controllers.js (hunk 1 above)
Comment (JavierJia) [inline neo/app/assets/javascripts/sidebar/controllers.js]: I don't know the details either. The [documentation](https://docs.angularjs.org/api/ng/service/$http#jsonp) says it has to be this name

Comment (JeremyLi28) [inline neo/app/assets/javascripts/sidebar/controllers.js]: Got it, thanks!

**Pull request merged**